python/tvm/micro/base.py

```diff
@@ -14,129 +14,156 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-"""Base definitions for micro."""
+"""Base definitions for MicroTVM"""
 
 from __future__ import absolute_import
 
 import logging
 import os
 import sys
+from enum import Enum
+from pathlib import Path
 
+import tvm
 from tvm.contrib import util as _util
 from tvm.contrib import cc as _cc
-
 from .._ffi.function import _init_api
 from .._ffi.libinfo import find_include_path
 
-SUPPORTED_DEVICE_TYPES = ["host", "openocd"]
+class LibType(Enum):
+    RUNTIME = 0
```

Comment:
document what is runtime and what is operator, is it possible to tell that by checking types of module?